### PR TITLE
docs: Use prerendered sponsors for README

### PR DESCRIPTION
**Pull request by mdjermanovic**
#### Prerequisites checklist

-   [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request?

Simplifies script that adds sponsors to READMEs.

#### What changes did you make? (Give an overview)

Updated `tools/update-readme.js` to use prerendered sponsors markdown from the eslint.org repo.

#### Related Issues

https://github.com/eslint/eslint.org/issues/619

### CODE REVIEW

File: tools/update-readme.js [hunk 1]
```diff
@@ -11,126 +11,46 @@
...
 
 		let newReadme = readme.replace(
 			/<!--sponsorsstart-->[\w\W]*?<!--sponsorsend-->/u,
-			formatSponsors(allSponsors),
-		);
-
-		newReadme = newReadme.replace(
-			/<!--techsponsorsstart-->[\w\W]*?<!--techsponsorsend-->/u,
-			formatTechSponsors(techSponsors),
+			`<!--sponsorsstart-->\n\n${allSponsors}\n<!--sponsorsend-->`,
```
Comment (harish-sethuraman) [inline tools/update-readme.js:53]: why do we have two new line in this repo alone? whilst we dont have it in other `eslint/js` and `eslint/eslint` repo?

Comment (mdjermanovic) [reply tools/update-readme.js:53]: Without the second one, Prettier would be reporting a formatting error there.

Comment (mdjermanovic) [reply tools/update-readme.js:53]: I also added another new line in the fetched content, for the same reason (https://github.com/eslint/eslint.org/pull/648).

File: tools/update-readme.js [hunk 2]
```diff
@@ -11,126 +11,46 @@
...
-    <!--techsponsorsend-->`;
+async function fetchSponsorsMarkdown() {
+	return got(SPONSORS_URL).text();
 }
 
 //-----------------------------------------------------------------------------
 // Main
 //-----------------------------------------------------------------------------
 
 (async () => {
```
Comment (amareshsm) [inline tools/update-readme.js:44]:
```suggestion

```

Comment (mdjermanovic) [reply tools/update-readme.js]: Removed in https://github.com/eslint/rewrite/pull/122/commits/0911a8f407fc88d43d052756187fb919f8638d29 now.

**Review by amareshsm — APPROVED** [review]
LGTM

**Review by nzakas — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**